Sources/VLCFirstStepsViewController.m

```diff
@@ -69,11 +69,7 @@ - (void)viewWillDisappear:(BOOL)animated
 - (void)updateTheme
 {
     self.view.backgroundColor = PresentationTheme.current.colors.background;
-    if (@available(iOS 13.0, *)) {
```

Comment:
now navigationBar changes color without this code (cause of extension) - and without lag!